packages/prisma/migrations/20220811234822_add_after_meeting_ends_trigger/migration.sql

```diff
@@ -1,5 +1,3 @@
 -- AlterEnum
 ALTER TYPE "WebhookTriggerEvents" ADD VALUE 'MEETING_ENDED';
```

Comment:
```suggestion
-- close the transaction to support the new enum value on pg11
COMMIT;
-- AlterEnum
ALTER TYPE "WebhookTriggerEvents" ADD VALUE 'MEETING_ENDED';
```

Do you think this suggestion would fix this? @CarinaWolli @simobasso

Reply:
I am running the migrations locally using a postgres 11 heroku database and for me, it only works with the changes I made. Even once adding the COMMIT I get the error message 'ERROR: ALTER TYPE ... ADD cannot run inside a transaction block'. @simobasso does it work for you when adding the COMMIT?